### ISSUE TITLE
deeplab  predict a blank segmentation masks

### ISSUE THREAD

**Issue by essalahsouad** [tensorflow/models#6982]
### System information
What is the top-level directory of the model you are using:
https://github.com/tensorflow/models/tree/master/research/deeplab
** Have I written custom code (as opposed to using a stock example script provided in TensorFlow):**
No
**OS Platform and Distribution (e.g., Linux Ubuntu 16.04):**ubuntu 18.04
**TensorFlow installed from (source or binary):**binary
**TensorFlow version (use command below):**1.13.1 CPU
**Bazel version (if compiling from source):**N/A
**CUDA/cuDNN version:**9.2/7.1
**GPU model and memory:**GeForce GTX 970M
- **Exact command to reproduce**:
`python deeplab / train.py \ 
--logtostderr \ 
--vis_split = "train" \ 
--model_variant = "xception_65" \ 
--atrous_rates = 6 \ 
--atrous rates = 12 \ 
--atrous rates = 18 \ --output_stride = 16 \ 
--decoder_output_stride = 4 \ 
--training_number_of_steps = 1000
--train_crop_size = 513 \ --train_batch_size = 1 \
--train_crop_size = 513 \
 --Fine_tune_batch_norm=False \ 
--Tf_initial_checkpoint = "./ Data / Init_models / Deelabv3_pascal_train_aug \ model.ckpt" 
--Initialize_last_layer = False \ 
--Last_layers_contain_logits_only = True \ `
**### Describe the problem**
Hi i trained deeplab model and does not predict anything just a black background, i don't know what's the problem 
### -training on custom dataset (iris data)
-data = RGB image + ( 0-1) label: 400 * 300 
-classe=2
-convert to record format: 
-training step: 
```
python deeplab / train.py \ 
--logtostderr \ 
--vis_split = "train" \ 
--model_variant = "xception_65" \ 
--atrous_rates = 6 \ 
--atrous rates = 12 \ 
--atrous rates = 18 \ --output_stride = 16 \ 
--decoder_output_stride = 4 \ 
--training_number_of_steps = 1000
--train_crop_size = 513 \ --train_batch_size = 1 \
--train_crop_size = 513 \
 --Fine_tune_batch_norm=False \ 
--Tf_initial_checkpoint = "./ Data / Init_models / Deelabv3_pascal_train_aug \ model.ckpt" 
--Initialize_last_layer = False \ 
--Last_layers_contain_logits_only = True \ 
...
```
==> Loss = 0.2 ～ 0.1 
-convert to .pb step 
python export_model.py \ --logtostderr \ -model_variant = "xception_65" \ --atrous_rates = 6 \ --atrous_rates = 12 \ --atrous_rates = 18 \ --output_stride = 16 
until this step everything looks fine 
-test step 
![1](https://user-images.githubusercontent.com/34586193/59139163-325d2200-8989-11e9-863d-262975cb43f4.JPG)
![2](https://user-images.githubusercontent.com/34586193/59139167-3b4df380-8989-11e9-8498-66123994a595.JPG)
![3](https://user-images.githubusercontent.com/34586193/59139176-4739b580-8989-11e9-807a-2599c3be6489.JPG)
![Capture](https://user-images.githubusercontent.com/34586193/59139184-528ce100-8989-11e9-90aa-bf49d244f11a.JPG)
would be nice if the @aquariusjay can provide some suggestions

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Exact command to reproduce

**Comment by essalahsouad** [comment]
@tensorflowbutler i updated my issue

**Comment by ajinkya933** [comment]
Make sure you mask your iris with RGB(1,1,1) and mask your background with RGB(0,0,0)

**Comment by VicXue** [comment]
@essalahsouad Hi, have you found out why this is happening?
